src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -321,20 +354,34 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
     if (Number(this._config![ev.target.key].value) === value) {
       return;
     }
+    this.setResult(ev.target.key, undefined);
     this.debouncedUpdate(ev.target, value);
   }
 
-  private _updateConfigParameter(target, value) {
+  private async _updateConfigParameter(target, value) {
     const nodeId = getNodeId(this._device!);
-    setNodeConfigParameter(
-      this.hass,
-      this.configEntryId!,
-      nodeId!,
-      target.property,
-      value,
-      target.propertyKey ? target.propertyKey : undefined
-    );
-    this._config![target.key].value = value;
+    try {
+      const result = await setNodeConfigParameter(
+        this.hass,
+        this.configEntryId!,
+        nodeId!,
+        target.property,
+        value,
+        target.propertyKey ? target.propertyKey : undefined
+      );
+      this._config![target.key].value = value;
+
+      this.setResult(target.key, result.status);
```

Comment:
We set the value in the config, and then set the result to config? I don't get this? I think we should combine these in the set result function?